src/stringobj.c

```diff
@@ -1906,26 +1906,16 @@ Obj FuncSplitString (
     return FuncSplitString( self, string, seps, wspace );
   }
   
-  /* reset SPLITSTRINGSEPS (in case of previous error) */
-  if (SPLITSTRINGSEPS[256] != 0) {
-    for(i=0; i<257; i++) SPLITSTRINGSEPS[i] = 0;
-  }
-  
   /* set SPLITSTRINGSEPS by setting positions of characters in rem to 1 */
   len = GET_LEN_STRING(seps);
   s = CHARS_STRING(seps);
-  SPLITSTRINGSEPS[256] = 1;
```

Comment:
Rather than using memset here (and on line 1918), just declare the arrays as ` = {0}`?

Also, maybe add in the commit message that the '256'th element was used to mark the array as dirty, just in case anyone ever wonders where the last element went (although I don't imagine anyone will!)